src/lib/github.js

```diff
@@ -20,7 +20,7 @@ module.exports = async function (info) {
       type: 'input',
       name: 'token',
       message:
-        'Provide a GItHub Personal Access Token (create a token at https://github.com/settings/tokens/new?scopes=repo)',
+        'Provide a GitHub Personal Access Token (create a token at https://github.com/settings/tokens/new?scopes=repo)',
```

Comment:
do you want to make a separate PR for this typo fix?

Reply:
Fixed in [357](https://github.com/semantic-release/cli/pull/357)